tests/validation/validate_kinetic.hpp

```diff
@@ -23,11 +28,11 @@ void run_kinetic_dt(nest::mc::cell& c, float t_end, nlohmann::json meta, const s
     };
 
     meta["sim"] = "nestmc";
-    meta["backend"] = LoweredCell::backend::name();
+    meta["backend"] = backend==backend_policy::use_multicore? "cpu" : "gpu";
```

Comment:
This feels fragile, but it is probably due to the transitional nature of the change. I would add an TODO?